app/src/main/java/com/greenfox/kalendaryo/LoginActivity.java

```diff
@@ -119,16 +119,15 @@ public void handleResult(GoogleSignInResult result) {
             final String userName = account.getDisplayName();
             final String userEmail = account.getEmail();
             ApiService apiService = RetrofitClient.getApi("backend");
-            sharedPref = PreferenceManager.getDefaultSharedPreferences(getApplicationContext());
-            String clientToken = sharedPref.getString("clienttoken", "");
-            apiService.postAuth(clientToken, new KalAuth(account.getServerAuthCode(), userEmail, userName)).enqueue(new Callback<KalUser>() {
-              
+            String clientToken = kalPref.getString("clienttoken");
```

Comment:
let's refactor this into a `kalPref.clientToken()` and while you're there let's have this `"clienttoken"` string extracted into a constant of `KalPref` class